src/generated/randr.rs

```diff
@@ -3265,7 +3266,6 @@ where
     let format_bytes = format.serialize();
     let mode_bytes = mode.serialize();
     let num_items_bytes = num_items.serialize();
-    assert_eq!(data.len(), (num_items as usize) * ((format as usize) / (8)), "Argument data has an incorrect length");
```

Comment:
Another assert (I guess this has the same underlying reason and will now stop reporting asserts in request functions, just search for `assert` in the diff to see if anything remains)

Reply:
Fixed, it seems that all asserts are now present